riscv-page-tables/src/page_table.rs

```diff
@@ -878,6 +959,43 @@ impl<'a, T: FirstStagePagingMode> FirstStageMapper<'a, T> {
         }
     }
 
+    /// Maps `num_pages` pages contiguously both physically and virtually, starting from
+    /// `start_vaddr`, which maps to `start_paddr`. The caller must guarantee that the `num_pages`
+    /// pages starting from `start_paddr` are safe to map in this page table.
+    ///
+    /// # Safety
+    ///
+    /// Don't create aliases.
+    pub unsafe fn map_contiguous_addr(
```

Comment:
personal preference naming nit: i'd cal this one `map_contiguous` and the one below `map_one`

Reply:
Ack.

Reply:
Done.